docs/DeveloperGuide.md

```diff
@@ -654,3 +655,28 @@ Team size: 5
 1. **Add more precise functionality to the `clean` command**. The `clean` command currently does not allow removal of 
 students who have graduated in the current year, as it can only detect the graduation year but not the month. We plan to 
 add support for storing a more specific graduation date, such that we can accurately remove students who have graduated immediately after their graduation.
+
+
+## **Appendix: Effort**
+
+### Overview
+
+As we have adapted AB3 for university dorm managers, our main efforts were in adding support for other necessary fields, enhancing the duplicate handling and data validation, and providing extra functions to streamline data saving, adding, updating and safeguard against mistakes.
```

Comment:
is better to phrase it as ".... saving, adding, updating and TO safeguard against mistakes."